src/components/AmbassadorCard.tsx

```diff
@@ -251,6 +255,22 @@ export default forwardRef(function AmbassadorCard(
             <p>{ambassador.native.text}</p>
           </div>
 
+          <div>
+            <h3 className={headingClass}>Lifespan</h3>
```

Comment:
```suggestion
            <h3 className={headingClass}>Species Lifespan</h3>
```
I think we should make it clear this is data on the species as a whole, not the specific ambassador